onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorPadding.cpp

```diff
@@ -95,6 +95,27 @@ class DmlOperatorPadding : public DmlOperator, public PaddingHelper
     }
 };
 
+void CALLBACK QueryPad(IMLOperatorSupportQueryContextPrivate* context, /*out*/ bool* isSupported)
+{
+    // At the time of writing this, DML_PADDING1_OPERATOR_DESC doesn't support negative element for 
```

Comment:
[minor wording]
"At the time of writing this, DML_PADDING1_OPERATOR_DESC doesn't..." -> "DML_PADDING1_OPERATOR_DESC doesn't..."

DML_PADDING1_OPERATOR_DESC never will support negative padding counts, now or later, because we can't change existing AP's. A DML_PADDING*2*_OPERATOR_DESC could though.

"doesn't support negative element for padding count" -> "doesn't support negative padding counts"

Because padding counts can come from either tensor elements or from attribute values.

Reply:
I didn't mean to say that PADDING1 will change. I agree my choice of words were not appropriate. Thank you for pointing this out.